packages/element-web-module-api/src/api/actions.ts

```diff
@@ -0,0 +1,18 @@
+/*
+Copyright 2025 New Vector Ltd.
+
+SPDX-License-Identifier: AGPL-3.0-only OR LicenseRef-Element-Commercial
+Please see LICENSE files in the repository root for full details.
+*/
+
+/**
+ * Exposes certain action that can be performed on element-web.
+ * @public
+ */
+export interface ActionsApi {
+    /**
+     * Open a room in element-web.
+     * @param roomId - id of the room to open
+     */
+    openRoom: (roomId: string) => void;
```

Comment:
Is this not already covered by the Navigation API? If not would it not fit better there?

Reply:
Sure, but aren't those *navigation* actions?

Reply:
Depends on what navigation means. I interpreted it as being related to URLs since that's what the two existing methods do. If you interpret navigation as in navigating through the UI, then yes, it makes sense to include this there.